spec/active_record/typed_store_spec.rb

```diff
@@ -958,19 +958,26 @@
 describe DirtyTrackingModel do
   it 'stores the default on creation' do
     model = DirtyTrackingModel.create!
+    model = DirtyTrackingModel.find(model.id)
```

Comment:
This `find` (or a `reload`) is required to exercise the fix from #95.  (i.e. Without a reload, this test would not have failed prior to #95 either.)